.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '6.0'
+          include-prerelease: True
+      - name: Setup .NET
```

Comment:
Are you setting up both .NET 6 and .NET 5 on purpose? 🤔

Reply:
YES. without both of them, the Project or SonarCloud tool won't build... It's weird but you can check previous runs of this pipeline without .NET 5 👇 

https://github.com/sde-sw/blazor-interview-app/runs/3523331996?check_suite_focus=true